### PR TITLE
QMapShack.app v1.3.1

### PR DESCRIPTION
**Pull request by kribe**

### CODE REVIEW

File: Casks/qmapshack.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'qmapshack' do
+  version '1.3.1'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/qmapshack.rb]: This needs to be set.

Comment (kribe) [inline Casks/qmapshack.rb]: Do you mean the SHA Check? According to documentation is that optional.

Comment (jawshooah) [inline Casks/qmapshack.rb]: It is used only in cases where an actual shasum is not useful. I.e. with `version :latest`, or if the upstream file is updated in-place without changing the URL.

**Comment by kribe** [comment]
added sha check sum.

**Comment by vitorgalvao** [comment]
Looks good. Can you please squash your commits?

**Comment by kribe** [comment]
commits are squashed now.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**